packages/server/src/routes/api/users/users-router.js

```diff
@@ -59,7 +61,22 @@ router.post(
   async (req, res, next) => {
     try {
       const validationResults = await validateEmailAndUsername(req.body);
-      res.status(200).send({ ...validationResults, emailExists: false });
+      res.status(200).send({ ...validationResults });
+    } catch (err) {
+      next(err);
+    }
+  },
+);
+
+router.patch(
+  "/:id",
```

Comment:
Should just get id from the token. Just add extractUserFromIdToken to middleware

Reply:
Not sure how to use that method for this lol, maybe add in another PR?